lib/vsc/__init__.py

```diff
@@ -1,37 +1,3 @@
-#!/usr/bin/env python
-##
-# Copyright 2015-2015 Ghent University
-#
-# This file is part of vsc-install,
-# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
-# with support of Ghent University (http://ugent.be/hpc),
-# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
-# the Hercules foundation (http://www.herculesstichting.be/in_English)
-# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
-#
-# http://github.com/hpcugent/vsc-install
-#
-# vsc-install is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Library General Public License as
-# published by the Free Software Foundation, either version 2 of
-# the License, or (at your option) any later version.
-#
-# vsc-install is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# GNU Library General Public License for more details.
-#
-# You should have received a copy of the GNU Library General Public License
-# along with vsc-install. If not, see <http://www.gnu.org/licenses/>.
-##
-"""
-Initialize vsc package.
-The vsc namespace is used in different folders allong the system
-so explicitly declare this is also the vsc namespace
-
-@author: Jens Timmerman (Ghent University)
-"""
-from pkgutil import extend_path
-
-# we're not the only ones in this namespace
-__path__ = extend_path(__path__, __name__)  #@ReservedAssignment
+# allow other packages to extend this namespace, zip safe setuptools style
+import pkg_resources
```

Comment:
fallback to `pkgutil.extend_path` in case `pkg_resources` is not available (backward compatibility, etc.)?